src/NuGetPackageVerifier/third-party.json

```diff
@@ -17,7 +17,7 @@
 
   "packages": {
     "Microsoft.ApplicationInsights": [ "2.4.0-beta3", "2.4.0-beta5" ],
```

Comment:
No one, but it's pulled in transitively via M.AI.AspNetCore. I've updated the version as well but I'm not sure if we should include this in the first place. Not too familiar with how this tool works.